wordle-web/pages/instructionsPage.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-app style="background: #064a80">
```

Comment:
This is really nice. I gave you guys a couple of extra points for the additional effort on this page!